### DISCUSSION TITLE
I got `An error occurred updating the repository, please try again later.`, when update repository.

### DISCUSSION THREAD

**Discussion by zong-zhe** [artifacthub/hub#3492] [Q&A] [answered]
Hi , I got an error when I update the repository into `Disabled`. 😲😲😲

<img width="652" alt="截屏2023-11-20 11 35 20" src="https://github.com/artifacthub/hub/assets/68977949/6186b7f0-d2f9-4dd3-91c5-bf7616c92b1e">

And there is another error, if I want to delete an useless repository.

![bbb](https://github.com/artifacthub/hub/assets/68977949/d687858b-c2b1-4e6e-8f05-e21f2bd7c672)

All these functions can be work in https://staging.artifacthub.io/

**Comment by tegioz** [comment, marked as answer]
Hi @zong-zhe

Can you try again? There were a couple of missing indexes in a database table, so those delete/update requests were taking longer than expected and timing out. It should be fine now 🙂

**Reply by tegioz** [reply]
Fixed by #3504
